web-tagging.html

```diff
@@ -0,0 +1,278 @@
+---
+layout: default
+title: Using OMERO.webtagging
+menu-id: l1-workflow2
+pdfs:
+    - web-tagging.pdf
+description: How to use the tools in OMERO.webtagging, Auto-Tagger and Tag Search to add tag annotations to objects and use them to search.
+version:
+    - yes
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#auto">Using Autotagging</a><br />
+<a href="#search">Using Tag Search</a><br />
+
+</p>
+</div><!-- End Page Navigation -->
+
+<div class="line-block">
+<div class="line"><br /></div>
+</div>
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
+
+<p>OMERO.webtagging is the umbrella for tools developed to enhance use of text annotations (tags) in OMERO. There are two tools at present, and either or both can be installed as part of the OMERO.web client.</p>
```

Comment:
And yes, I do now see this is from the docs page and I've just updated it there ;)